src/styles/theme.scss

```diff
@@ -187,7 +218,7 @@ h5 {
 .button:active,
 .button:focus,
 .button:hover {
-  background-color: $purple-light;
+  background-color: $pink-light;
```

Comment:
these might be better defined as theme colours, rather than using the colour name.